src/Toolbar.js

```diff
@@ -65,8 +65,7 @@ export const getStartOfSession = () => {
   const map = {
     '2022S1': new Date('2022-02-19Z21:00:00'), // 8AM 21 Feb in GMT
   }
```

Comment:
is it evil to pull a
```suggestion
  return {
    '2022S1': new Date('2022-02-19Z21:00:00'), // 8AM 21 Feb in GMT
  }?.[year + session]
```